.gitignore

```diff
@@ -66,6 +66,9 @@ node_modules/
 codekit-config.json
 .pycharm_helpers/
 
+.project
+.pydevproject
```

Comment:
I just assumed I could add these give that there were some for pycharm as well, but I can remove them